Python/chapter04/lib/graph_search.py

```diff
@@ -0,0 +1,143 @@
+import unittest
```

Comment:
is this file used by the solution? if not, it can be removed

Reply:
Not used directly by the solution.  Going to remove.  Moved this code to a separate PR to make it easier for me to reference commonly used code

Reply:
File removed here:  59bceeefc2449efb37ef2aa5a948e234e2b8ec66